peg/kotlin_post.h

```diff
@@ -0,0 +1,164 @@
+/*
+ *   Copyright (c) 2019 Masatake YAMATO
+ *   Copyright (c) 2019 Red Hat, Inc.
+ *
```

Comment:
I didn't write this.

Reply:
Oh, sorry. Stupid copy&paste error :facepalm: